seeds/_sector/dex/curvefi_optimism_base_trades_seed.csv

```diff
@@ -1,18 +1,21 @@
+optimism,curve,1,2023-01-30,0x72f451639593a62c0ec916d20de947b11d15f2ae5075452ddf08d68f77b54e87,8,0x8c6f28f2f1a3c87f0f938b96d27520d9751ec8d9,0x94b008aa00579c1307b0ef2c499ad98a8ce58e58,71362505,99875991930621765,98360937855263771
+optimism,curve,1,2023-01-31,0xa70d826c5b3b5afbe9585e15f1e58c75cbcc1359784d48de48b9d025270985d9,70,0x7f5c764cbc14f9669b88837ca1490cca17c31607,0x94b008aa00579c1307b0ef2c499ad98a8ce58e58,71379460,32037157,32036492
+optimism,curve,1,2023-01-31,0x2a4ff09a9d921f631e4e14a61fc6f7fd7997691ed40acac415c0988fe93f9f96,39,0x7f5c764cbc14f9669b88837ca1490cca17c31607,0x94b008aa00579c1307b0ef2c499ad98a8ce58e58,71426874,16006302,16006705
+optimism,curve,1,2023-01-30,0xf14d489c5cb44772b1082e7589781fac60d0f559fa2999844b2cbb782d434443,75,0x7f5c764cbc14f9669b88837ca1490cca17c31607,0xda10009cbd5d07dd0cecc66161fc93d7c9000da1,71327787,8002330,8002495905917787257
+optimism,curve,1,2023-01-30,0x2615836b17d0b9a467f187a84c497233ee3a815a0ad341d70b63e28d2c6c98df,159,0x7f5c764cbc14f9669b88837ca1490cca17c31607,0xda10009cbd5d07dd0cecc66161fc93d7c9000da1,71331295,128283656,128286430267516232325
+optimism,curve,1,2023-01-30,0x4569f2998b9f00eea1c372d82a36fa32b7aa7ca9fba7378bf63756614e2733e9,3,0x94b008aa00579c1307b0ef2c499ad98a8ce58e58,0xda10009cbd5d07dd0cecc66161fc93d7c9000da1,71338934,255877053,255914112076597632185
+optimism,curve,1,2022-11-12,0xcea94039dcf9ce75c6081f0ed26fc3f6b9193036f901372b002edc67f960a711,9,0x94b008aa00579c1307b0ef2c499ad98a8ce58e58,0x2e3d870790dc77a83dd1d18184acc7439a53f475,37329347,1012,1017988696243052
+optimism,curve,1,2022-05-20,0xf0a56683a1f5c98f734da37362976c19ced781ea2ca1168c203d605593689308,3,0x94b008aa00579c1307b0ef2c499ad98a8ce58e58,0x7f5c764cbc14f9669b88837ca1490cca17c31607,8659786,112531495,106894737
+optimism,curve,1,2022-05-16,0x1daf9b0e5b53bbb0cf2cc82fc56ea4b03e4b04642326c4c34ee5034edcc419a0,5,0x2e3d870790dc77a83dd1d18184acc7439a53f475,0x94b008aa00579c1307b0ef2c499ad98a8ce58e58,8429446,2028066389647480735900,2002456704
+optimism,curve,1,2022-05-11,0x9e35d672d4202923c30f0e65eeaae4b7e922600b2715dacde56a98cef3903062,5,0x94b008aa00579c1307b0ef2c499ad98a8ce58e58,0x2e3d870790dc77a83dd1d18184acc7439a53f475,7910924,1002932210,1001247450302501874441
+optimism,curve,1,2022-05-14,0xa3178cf9778f11a12c896e7967a38ab72b2ca043e2f569dc56e7fe58de6d7995,3,0x2e3d870790dc77a83dd1d18184acc7439a53f475,0xfb21b70922b9f6e3c6274bcd6cb1aa8a0fe20b80,8234971,6057514570698228151262,30000000000
+optimism,curve,1,2023-09-05,0x9e32c44a4f5dae97fe72f8ff3aa52707180beaafb85fc0823e4e273450667ca5,490,0x8c6f28f2f1a3c87f0f938b96d27520d9751ec8d9,0x94b008aa00579c1307b0ef2c499ad98a8ce58e58,109170536,4000860630461955786,3916237747392440611
+optimism,curve,1,2023-09-05,0x9e32c44a4f5dae97fe72f8ff3aa52707180beaafb85fc0823e4e273450667ca5,478,0x8c6f28f2f1a3c87f0f938b96d27520d9751ec8d9,0x94b008aa00579c1307b0ef2c499ad98a8ce58e58,109170536,4000925723992149027,3916301433007904588
```

Comment:
Same goes to `3916301433007904588` here and elsewhere, in this event the token involved is `3.916301433007904588 Curve.fi DAI/USDC/USD`. You can have a check.